KitePrintSDK/src/main/java/ly/kite/print/PostcardPrintJob.java

```diff
@@ -72,7 +70,7 @@ private static String getStringOrEmptyString(String val) {
 
     private JSONObject getJSON() throws JSONException{
         JSONObject json = new JSONObject();
```

Comment:
This postcard print job JSON blob has been radically simplified in recent months -- see the iOS code here: https://github.com/OceanLabs/iOS-Print-SDK/blob/master/Kite%20SDK/PSPrintSDK/OLPostcardPrintJob.m#L85-L113

And REST docs here: https://www.kite.ly/docs/#ordering-postcards

We'll need to update this `getJSON` method to reflect the new interface. Not a dependency for the merge, just bringing it to your attention as it was here. I'll add a card to trello to capture this task